### PR TITLE
SR1 Fixes into SRFix branch

### PR DESCRIPTION
**Pull request by zixuan97**
fix: rename last_name into lastName, and first_name into firstName (camelCase standardisation from FE and BE) 
fix: validator checks on edge case of undefined (will need to match backend for camel case, otherwise code breaks)

### CODE REVIEW

File: src/pages/account/CreateNewUser.tsx [hunk 1]
```diff
@@ -208,10 +211,15 @@ const CreateNewUser = () => {
                 </Button>
                 <Button
                   disabled={
-                    !validator.isEmail(newUser.email!) ||
-                    validator.isEmpty(newUser.last_name!) ||
-                    validator.isEmpty(newUser.first_name!) ||
-                    validator.isEmpty(newUser.role!)
+                    newUser.email === undefined ||
```
Comment (danielonges) [inline src/pages/account/CreateNewUser.tsx:214]: can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) and [null coalesce operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) to make this less verbose

File: src/components/account/accountTypes.ts [hunk 2]
```diff
@@ -1,22 +1,9 @@
+import { UserRole } from 'src/models/types';
+
 export const roles = [
```
[the remark this answers is not in the document]
Comment (zixuan97) [reply src/components/account/accountTypes.ts]: Switched to enum.

File: src/pages/account/CreateNewUser.tsx (hunk 1 above)
Comment (zixuan97) [reply src/pages/account/CreateNewUser.tsx]: Edited!

**Review by leeleonard98 — APPROVED** [review]

**Review by danielonges — APPROVED** [review]
lgtm

**Pull request merged**